pkgs/leak_tracker/lib/src/leak_tracking/leak_tracker_model.dart

```diff
@@ -95,7 +95,7 @@ class LeakTrackingConfiguration {
     this.checkPeriod = const Duration(seconds: 1),
     this.disposalTimeBuffer = const Duration(milliseconds: 100),
     this.leakDiagnosticConfig = const LeakDiagnosticConfig(),
-    this.gcCountBuffer = defaultGcCountBuffer,
+    required this.gcCountBuffer,
```

Comment:
We should mention in the changelog that this is now required.

Reply:
(and / or ensure that the changes mentioned there cover user-facing ones, like api changes)

Reply:
reverted